codespell_lib/data/dictionary.txt

```diff
@@ -15594,6 +15682,7 @@ wirtes->writes
 wirting->writing
 wit->wit, with,
 witdh->width
+with out->without
```

Comment:
Codespell can't currently handle this. Please remove it and add it to the list in https://github.com/codespell-project/codespell/issues/74 .

Reply:
done